src/seer/db.py

```diff
@@ -255,6 +255,24 @@ class DbPrIdToAutofixRunIdMapping(Base):
     )
 
 
+class DbPrContextToUnitTestGenerationRunIdMapping(Base):
+    __tablename__ = "codegen_unit_test_generation_pr_context_to_run_id"
+    id: Mapped[int] = mapped_column(Integer, primary_key=True)
+    provider: Mapped[str] = mapped_column(String, nullable=False)
+    owner: Mapped[str] = mapped_column(String, nullable=False)
+    pr_id: Mapped[int] = mapped_column(BigInteger, nullable=False)
+    repo: Mapped[str] = mapped_column(String, nullable=False)
+    run_id: Mapped[int] = mapped_column(
+        ForeignKey(DbRunState.id, ondelete="CASCADE"), nullable=False
+    )
+    iterations = mapped_column(Integer, nullable=False, default=0)
+    original_pr_url: Mapped[str] = mapped_column(String, nullable=False)
+    __table_args__ = (
+        UniqueConstraint("provider", "pr_id", "repo", "owner", "original_pr_url"),
+        Index("ix_autofix_repo_owner_pr_id", "owner", "repo", "pr_id", "original_pr_url"),
```

Comment:
```suggestion
        Index("ix_unittest_repo_owner_pr_id", "owner", "repo", "pr_id", "original_pr_url"),
```
index still named autofix, you'll need to regenerate the migration